config/java_memory_assistant.yml

```diff
@@ -0,0 +1,33 @@
+# Encoding: utf-8
+# Cloud Foundry Java Buildpack
+# Copyright 2013-2017 the original author or authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+version: 0.1.0
+repository_root: https://raw.githubusercontent.com/SAP/java-memory-assistant/repository
```

Comment:
So far only SAP-private internal repos. Since it is an agent and not a library, I was thinking of just relying on GitHub releases. But if a good use case shows up, I think I could arrange publishing on something like Maven Central or a Sonatype Nexus of choice.